defaults/main.yml

```diff
@@ -14,8 +14,8 @@ ssl_cert_path: /etc/pki/tls/certs
 ssl_key_path: /etc/pki/tls/private
 
 # UI
-cas_screen_welcome_instructions: Enter your Username and Password
-cas_screen_welcome_security: For security reasons, please Log Out and Exit your web browser when you are done accessing services that require authentication!
+cas_screen_welcome_instructions: Login with your OUNetID (4x4)
```

Comment:
I have set the variables in vagrant infrastructure in the host_vars now. In place of the hard coded version is the original default values.